tests/integration-tests/tests/update/test_update/test_dynamic_file_systems_update/storage-stack.yaml

```diff
@@ -0,0 +1,238 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: >-
+  This template creates EFS, FSX Lustre, FSX Ontap, FSX Open ZFS storage that can be attached to a cluster.
+Parameters:
+  SubnetId0:
+    Type: String
```

Comment:
Minor non blocking comment (I've already added a comment on this, resolved but not addressed).
Parameters are missing a description.

Reply:
Yes, I changed the parameter names to be more descriptive instead of adding a description to every parameter.